### ISSUE TITLE
Add aria label option to Read More links and buttons

### ISSUE THREAD

**Issue by chadbrokaw-exygy** [bloom-housing/ui-components#22]
Dahlia has conducted an audit on the current React application and one of the things that came up was the lack of alt text on the links that can be optionally attached onto the PreferenceList component. Another was that we cannot attach alt text onto the Read More links on Expandable Text.

Our proposed solution is to alter the ListPreferenceLink interface so that it accepts alt text. Further, we propose to add an aria label option onto the strings prop in ExpandableText.

Acceptance Criteria in DAHLIA:

- Alt text can be added to the link element on the Preference List
- Alt text can be added onto the Expandable Text button

![Screenshot 2023-02-10 at 1.37.16 PM.png](https://images.zenhubusercontent.com/63e1277f5e22ef5e3b2f9505/5bd7881b-eaff-44df-8075-0891b69bd044)
